kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/SessionTrackerCookie.java

```diff
@@ -41,15 +41,18 @@ private SessionTrackerCookie() {
     public static void setIfNeeded(HttpSession session,
             HttpServletRequest request, HttpServletResponse response) {
         Optional<Cookie> clusterKeyCookie = getCookie(request);
-        if (!clusterKeyCookie.isPresent()) {
+        if (clusterKeyCookie.isEmpty()) {
             String clusterKey = UUID.randomUUID().toString();
             session.setAttribute(CurrentKey.COOKIE_NAME, clusterKey);
-            response.addCookie(new Cookie(CurrentKey.COOKIE_NAME, clusterKey));
+            Cookie cookie = new Cookie(CurrentKey.COOKIE_NAME, clusterKey);
+            cookie.setHttpOnly(true);
+            cookie.setPath("/");
```

Comment:
I don't remember if the path is relative to the webapp `context-path` or if it is relative to the container?
If it is the first one, it seems OK to me; otherwise the values should be computed based on `request.getContextPath()`.
@tamasmak do you mind making some test?

Reply:
Made a test using a container. I got these values and cookies:

![Screenshot 2024-02-29 at 09 55 12](https://github.com/vaadin/kubernetes-kit/assets/11977763/43fdabbe-eda8-4cf2-bd4e-4727a87c37be)
  ```
2024-02-29T08:12:11.513Z DEBUG SessionTrackerFilter: request.getContextPath()=''
2024-02-29T08:12:11.514Z DEBUG SessionTrackerFilter: request.getRequestURI()='/'
2024-02-29T08:12:11.514Z DEBUG SessionTrackerFilter: request.getRequestURL()='http://localhost:8080/'
2024-02-29T08:12:11.514Z DEBUG SessionTrackerFilter: request.getServletPath()='/'
2024-02-29T08:12:11.514Z DEBUG SessionTrackerFilter: request.getServerName()='localhost'
2024-02-29T08:12:11.514Z DEBUG SessionTrackerFilter: request.getServerPort()='8080'
  ```
  It seems to me that it is relative to the webapp `context-path`.
  @mcollovati What do you think?

Reply:
@tamasmak did you tested also setting a different context-path, e.g. `http://localhost:8080/myapp/'?

Reply:
Your assumption was right. The `context-path` is relative to the container. Setting it to
```
server:
  servlet:
    context-path: /demo
```
and using `cookie.setPath("/");` results in

![Screenshot 2024-02-29 at 10 38 57](https://github.com/vaadin/kubernetes-kit/assets/11977763/d6a8e71e-dd4d-47a9-9877-77de84c026f9)
Using the `cookie.setPath(request.getContextPath());` results in

![Screenshot 2024-02-29 at 10 35 17](https://github.com/vaadin/kubernetes-kit/assets/11977763/3efbc979-2145-4fbb-8a72-cd2242897618)
I committed a change to use the context-path.